package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hmrc-frontend",
-  "version": "3.1.1",
+  "version": "3.2.0",
```

Comment:
@tomshaw1 I spotted one more small thing, when you change the version number in here you need to do an npm install I think so that the version in package-lock.json increments as well.

Reply:
@oscarduignan gotcha, done that now too thank you